src/CommandingOfficers/Meridian.java

```diff
@@ -59,17 +60,17 @@ public Commander create(GameScenario.GameRules rules)
 
   final VehicularCharge myVehicularCharge = new VehicularCharge(this);
   private static final int POST_REFRESH_STAT_ADJUSTMENT = -25;
+  final UnitModel tank, arty;
+  final int costShift;
 
   public Meridian(GameScenario.GameRules rules)
   {
     super(coInfo, rules);
 
     // Meridian's basic tanks and arty cost the same
-    UnitModel tank = getUnitModel(UnitModel.ASSAULT);
-    UnitModel arty = getUnitModel(UnitModel.SIEGE);
-    int costShift = (tank.getCost() - arty.getCost())/2;
-    tank.costShift -= costShift;
-    arty.costShift += costShift;
+    tank = getUnitModel(UnitModel.ASSAULT);
+    arty = getUnitModel(UnitModel.SIEGE);
+    costShift = (getCost(tank) - getCost(arty))/2;
```

Comment:
Would it be better to handle this as a filtered unit modifier?

Reply:
Thought about that. I even wrote the class to filter by a single `UnitModel`.
I feel like it's cleaner to just have the CO do it, in this case - either I'm making two filters for two separate operations, or I'm making a new filter type *and* a new custom modifier type that both changes cost and adds an action.

Also, I'm slightly disposed towards avoiding more modifier instances - they're not free since they end up no-oping on basically any calculation you want to do with a unit.

Reply:
The current approach of using a single immutable shared `UnitModel` does lend itself to more modifiers though - thoughts on using an immutable model per CO? Not sure it would actually be better, but there might be some benefits.

Reply:
It does, though if we're super worried about it we can just make sure that the D2D ends up in the `Commander` directly. The reason I didn't do so for Strong/Venge/etc is basically laziness - it's just as clear and fewer lines to do it this way.

As for "immutable" models per CO, that's... basically what we have already, as of merging the last PR?
If we modify the model directly for the D2D, then it's not really immutable.
If we *don't* modify the model directly for the D2D, then why not make the models common to all COs?

The benefit of this design is that we don't need an enum to map unit types across COs - everyone just uses the same models, so you can check equality/etc safely across CO lines.

Reply:
> It does, though if we're super worried about it we can just make sure that the D2D ends up in the Commander directly.

We could. Though the Commander is a UnitModifier itself anyway, so... potato potato.

> As for "immutable" models per CO, that's... basically what we have already, as of merging the last PR?

Maybe. It was the initial vision for unit models, at any rate, and it would avoid some of the ugliness of e.g. not knowing how far your units can move without doing some work. On the other hand, it's nice to have common models across the board, and that wasn't possible in the initial incarnation.

Reply:
The Commander is a UnitModifier, yes, but it's always going to be there, therefore it's always going to be less processing overall to put functions into it rather than adding them to `Commander`'s list.
> Maybe. It was the initial vision for unit models, at any rate

I can believe that, but me-of-yesteryear obviously didn't really get that memo, hah
> it would avoid some of the ugliness of e.g. not knowing how far your units can move without doing some work

I don't think that's true, actually. If the models are immutable, then Strong's inf movement boost during powers is going to have to go into a `UnitModifier` and you're gonna have to check for that anyway (and do it everywhere, because inconsistency is bad).

Reply:
> then Strong's inf movement boost during powers is going to have to go into a UnitModifier

Nah, it could just be a remodel. That's not necessarily a cleaner/better approach in all respects, but it would certainly work.

Reply:
That's a fair point, I suppose. It wouldn't support the tags system I want without some cleverness, though.